src/blocks/social-profiles/block.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Is the block.json on this block is registered server-side? I noticed that core Gutenberg does not have block.json files for their server-side blocks (Latest Posts/Comments).

Reply:
You are correct that core blocks which are registered server side do **not** include block.json. In this case we were only using them to import the name of the block. This has been replaced with a string and the block.json has been removed in my latest commits.